schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -10,12 +10,14 @@ product_selection:
   - installation/product_selection/install_SLES
 license_agreement:
   - installation/licensing/accept_license
+configure_dasd_disks: []
```

Comment:
in s390x kvm we don't configure dasd disks, can be drop.

Reply:
fixed.